src/components/MenuItem.tsx

```diff
@@ -514,6 +514,7 @@ function MenuItem(
                                         ...(Array.isArray(wrapperStyle) ? wrapperStyle : [wrapperStyle]),
                                         !focused && (isHovered || pressed) && hoverAndPressStyle,
                                         shouldGreyOutWhenDisabled && disabled && styles.buttonOpacityDisabled,
+                                        isHovered && !pressed && styles.hoveredComponentBG
```

Comment:
NAB but let us add a trailing `,` so that when further styles are added as part of other PRs they don't have to bother about this. 
```suggestion
                                        isHovered && !pressed && styles.hoveredComponentBG,
```

Reply:
Sure